jedi/evaluate/recursion.py

```diff
@@ -3,8 +3,8 @@
 must stop recursions going mad. Some settings are here to make |jedi| stop at
 the right time. You can read more about them :ref:`here <settings-recursion>`.
 
-Next to :mod:`jedi.evaluate.cache` this module also makes |jedi| not
-thread-safe. Why?  ``execution_recursion_decorator`` uses class variables to
+Next to the internal ``jedi.evaluate.cache`` this module also makes |jedi| not
+thread-safe, because ``execution_recursion_decorator`` uses class variables to
```

Comment:
The module `jedi.evaluate.cache` is not documented (included in the docs), therefore this does not get linked - I've added the "internal" there assuming that non-documented means "internal".

Reply:
Makes sense.